docs/section19/19-7.md

```diff
@@ -0,0 +1,87 @@
+    - Sum OA CFM across proposed schedule to get the total for comparison purposes in the rule assertions: `modeled_proposed_total_zone_min_OA_CFM = sum(aggregated_min_OA_schedule_across_zones_p)`  
+
+
+
+    - **Rule Assertion:** 
+    - Case 1: If the modeled baseline and proposed minimum OA cfm values are identical then pass: `if OA_CFM_schedule_match == true: outcome = "PASS"`  
+    - Case 2: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and a zone served by the HVAC in the baseline has demand controlled ventilation modeled in the proposed (Exception 1) then the outcome is UNDETERMINED: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and was_DCV_modeled_baseline == false and was_DCV_modeled_proposed == true: outome = "UNDETERMINED" and raise_message "For <insert hvac.id> the baseline modeled minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears as though G3.1.2.5 Exception 1 may be applicable. A manual check for this exception is recommended otherwise fail."`  
+    - Case 3: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and a zone served by the HVAC in the baseline has an air distribution effectivenss of greater than 1 in the proposed (Exception 2) then the outcome is UNDETERMINED: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and zone_air_distribution_effectiveness_greater_than_1 == true: outome = "UNDETERMINED" and raise_message "For <insert hvac.id> the modeled baseline minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears as though G3.1.2.5 Exception 2 may be applicable. A manual check for this exception is recommended otherwise fail."`  
+    - Case 4: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and the hvac system only serves lab spaces (Exception 4) the outcome is UNDETERMINED: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and hvac_system_serves_only_labs == true: outome = "UNDETERMINED" and raise_message "For <insert hvac.id> the modeled baseline minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears as though G3.1.2.5 Exception 4 may be applicable. This exception states that for baseline systems serving only laboratory spaces that are prohibited from recirculating return air by code or accreditation standards, the baseline system shall be modeled as 100% outdoor air. A manual check for this exception is recommended otherwise fail."`  
```

Comment:
I think we need another RDS to check that baseline is 100% OA

Reply:
Need to address